### DISCUSSION TITLE
Paying bills for future period?

### DISCUSSION THREAD

**Discussion by HStep20** [firefly-iii/firefly-iii#5217] [Firefly III]
I get paid on the 31st (or latest weekeday), and my rent is due on the 1st. I like to pay rent the day I get paid, since its the 'top priority bill'. This is my first month using Firefly, so maybe I'm just misunderstanding the bill process. 

My period is set to 1 month, and I submit a bill transaction for 'Rent' on the 29th. It ties the bill payment to this period (October), instead of the next period that is coming up (November). If I go check the bills for next period, it will say that Rent is coming up in 3 days for the next period, even though its paid. 

Is the best solution to just mark the transaction date as November 1st, or are you able to 'future pay' bills for an upcoming period?

**Comment by JC5** [comment]
The first, that marker will always look at the current period

**Reply by th3guard1an** [reply]
Isn't there a way for the "overfill" to move unto the next month? I have a set ammount I need to pay monthly, but sometimes I pay a month in advance, but not always. Is there any way I can graph this?

**Reply by JC5** [reply]
No. You could store it as a separate transaction though.
